packages/libs/react-ui/src/components/NavHeader/NavHeaderNavigation.tsx

```diff
@@ -21,13 +21,27 @@ export const NavHeaderNavigation: FC<INavHeaderNavigationProps> = ({
     useGlow();
 
   useEffect(() => {
-    if (activeHref !== _activeHref) setActiveHref(activeHref);
+    if (activeHref !== _activeHref) {
+      setActiveHref(activeHref);
+    }
   }, [activeHref]);
 
   return (
     <NavHeaderNavigationContext.Provider
       value={{ setGlowPosition, activeHref: _activeHref, setActiveHref }}
     >
+      <div
+        role="none"
+        className={glowClass}
+        style={{
+          opacity: !glowX ? 1 : 0,
+          left: '-6rem',
+          transform: `translateX(${glowX}px)`,
+          transitionDuration: `${animationDuration}ms`,
+        }}
+      >
+        <NavGlow />
+      </div>
```

Comment:
Instead of adding an additional glow, can we make the one we already have start at this position?